user_management/views/menuitem.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+    <data>
+        <menuitem name="User Management"
+                  id="menu_eobs_category_user_management"
+                  parent="liveobs_core.menu_liveobs_root"
+                  groups="nh_clinical.group_nhc_ward_manager,base.user_root"/>
```

Comment:
Need to add group `nh_clinical.group_nhc_admin` otherwise this user won't see the 'User Management' Category under the LiveObs Root menu.